db/migrate/20240605143118_add_published_at_to_planning_applications.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddPublishedAtToPlanningApplications < ActiveRecord::Migration[7.1]
+  def change
+    add_column :planning_applications, :published_at, :datetime
+
+    up_only do
+      # https://en.wikipedia.org/wiki/Time_immemorial
+      PlanningApplication.where(make_public: true).update_all(published_at: Time.zone.local(1189, 7, 6, 0))
```

Comment:
I'm a bit wary of having this on production data.
Is there a more sensible proxy we could use or only run this for staging.
Not sure we'd have many `make_public: true` cases on prod with this tbh but maybe worth querying to see beforehand

Reply:
My thinking was that this needed to not be null (so that `make_public?` returns true), but also shouldn't be possible to mistake for any real date, so a date long in the past seemed to make the most sense. That way we can tell that it's public but see that we also don't know the real date that it was marked public.

Reply:
Yeah maybe just the validation date could potentially be better

Reply:
With the scope you added, we'd need to set a date on here anyway